Airports/EGAA/Freetext.txt

```diff
@@ -0,0 +1,29 @@
+;EGAA SRA Advisory Altitudes
+N054.36.10.777:W006.24.59.561:EGAA SRA 07 altitudes:*7nm-2500ft'
```

Comment:
```suggestion
N054.36.10.777:W006.24.59.561:EGAA SRA 07 altitudes:7-2500'
```

Reply:
Ill get on that now. The misc should be deleted unless Ive missed a step...Ill double check.

I deliberately made the txt size larger in the screen shot to clearly show the txt.

Cheers!